ethcore/src/state.rs

```diff
@@ -248,7 +251,13 @@ impl State {
 	/// Commit accounts to SecTrieDBMut. This is similar to cpp-ethereum's dev::eth::commit.
 	/// `accounts` is mutable because we may need to commit the code or storage and record that.
 	#[cfg_attr(feature="dev", allow(match_ref_pats))]
-	pub fn commit_into(trie_factory: &TrieFactory, db: &mut HashDB, root: &mut H256, accounts: &mut HashMap<Address, Option<Account>>) {
+	pub fn commit_into(
+		trie_factory: &TrieFactory,
+		db: &mut HashDB,
+		root: &mut H256,
+		accounts: &mut HashMap<Address,
+		Option<Account>>
```

Comment:
it could, but the line stands at something like 130 characters plus a tab. it seems cleaner to indent it in some way.